server/routes/projects/remix-bar.js

```diff
@@ -0,0 +1,27 @@
+var moment = require("moment");
```

Comment:
can you convert this file to ES6? We're trying to slowly convert Thimble towards it. For this file, I think the only change that needs to be made is converting all the `var` to `let`. Oh and adding "use strict" at the beginning.